server/channels/api4/job_test.go

```diff
@@ -314,9 +314,9 @@ func TestDownloadJob(t *testing.T) {
 
 	// Now we stub the results of the job into the same directory and try to download it again
 	// This time we should successfully retrieve the results without any error
-	filePath = "./data/export/" + job.Id + ".zip"
-	mkdirAllErr = os.MkdirAll(filepath.Dir(filePath), 0770)
-	require.NoError(t, mkdirAllErr)
+	filePath = filepath.Join(*th.App.Config().FileSettings.Directory, "export/", job.Id+".zip")
```

Comment:
We don't need the slash after "export" right?

Reply:
Nope, removed.